Source/SLLogger.h

```diff
@@ -0,0 +1,76 @@
+//
+//  SLLogger.h
+//  LoggingDemo
+//
+//  Created by 利辺羅 on 2014/12/09.
+//  Copyright (c) 2014年 SLF4C. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol SLLogger, SLMarker;
+
+typedef NS_ENUM(NSUInteger, SLLevel)
+{
+    SLLevelOff = 0,
+    SLLevelVerbose,
+    SLLevelDebug,
+    SLLevelInfo,
+    SLLevelWarning,
+    SLLevelError
+};
```

Comment:
About future extensibility, couldn't we use a log levels configuration file (i.e. plist)? That way, regular users will have a prebuilt version of the file and the others will be able to change it without changing the actual code.